tests/envs/test_make.py

```diff
@@ -52,13 +53,14 @@ def test_make():
 
 
 def test_make_deprecated():
-    with pytest.raises(
-        gym.error.Error,
-        match=re.escape(
-            "Environment version v0 for `Humanoid` is deprecated. Please use `Humanoid-v4` instead."
-        ),
-    ):
-        gym.make("Humanoid-v0", disable_env_checker=True)
+    with warnings.catch_warnings(record=True):
```

Comment:
Don't we want to check whether the caught warnings can be safely ignored? As we do in /tests/envs/test_envs.py?